README.md

```diff
@@ -163,6 +163,8 @@ iis_virtual_directory { 'vdir':
 
 Allows creation of a new IIS Application and configuration of application parameters.
 
+The iis_application type uses a composite namevar for applicationname and sitename to uniquely identify a declaration. To use this successfully, put both the sitename and the applicationname in the title. Puppet will build the catalog using the composite of the two values, while still using the correct value for the applicationname when creating the IIS application.  It requires a `\` in between the `sitename` and `applicationname`, for example, `iis_application { '#{@site_name}\\#{@app_name}'`. 
```

Comment:
Could the variables be placed in code blocks to improve readability? Especially since they're all lower case.

Reply:
Out of scope for this PR. The commit that caused this to show up in the diff has been rebased out of this PR.